lib/util/player_configuration.js

```diff
@@ -94,6 +94,12 @@ shaka.util.PlayerConfiguration = class {
       ignoreDuplicateInitData: !shaka.util.Platform.isTizen2(),
     };
 
+    if (shaka.util.Platform.isXboxOne() ||
+        shaka.util.Platform.isPS4() ||
+        shaka.util.Platform.isPS5()) {
```

Comment:
AFAIK PlayStation also supports Widevine.

Reply:
Only PS5, https://castlabs.com/resources/drm-comparison/ . I updated the code! Thanks!